client/landing/stepper/declarative-flow/internals/steps-repository/sensei-purpose/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { TextControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React, { useEffect } from 'react';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { SenseiStepContainer } from '../components/sensei-step-container';
+import { SenseiStepProgress } from '../components/sensei-step-progress';
+import PurposeItem from './purpose-item';
+import type { Step } from '../../types';
+import './style.scss';
+
+const wait = ( ms: number ) => new Promise( ( res ) => setTimeout( res, ms ) );
```

Comment:
I don't think that defining a function for a single statement makes things clearer.

Reply:
This is a common little helper to make a promise-based timeout, I think it's clearer when using it just as `await wait( 500 );`

Reply:
Yeah, it's just that as I read the code, it takes me more time to navigate to the function to see what it does than it would take me to read a single statement.

In any case, I don't consider this a blocker.